<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_widget</class>
 <widget class="QWidget" name="login_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="main_widget" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>311</width>
     <height>291</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>161</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="login_button">
       <property name="minimumSize">
        <size>
         <width>111</width>
         <height>61</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(250, 233, 143);</string>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="register_button">
       <property name="minimumSize">
        <size>
         <width>111</width>
         <height>61</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(250, 233, 143);</string>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit_button">
       <property name="minimumSize">
        <size>
         <width>111</width>
         <height>61</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(250, 233, 143);</string>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>511</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/title_generate.jpg);</string>
   </property>
  </widget>
  <widget class="QWidget" name="user_register" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>391</width>
     <height>291</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>360</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="user_form_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="id_2">
         <property name="text">
          <string>    用户名  ：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="register_id">
         <property name="minimumSize">
          <size>
           <width>213</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>213</width>
           <height>23</height>
          </size>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>请输入用户名</string>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::LogicalMoveStyle</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="password_2">
         <property name="text">
          <string>     密码   ：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="register_password">
         <property name="minimumSize">
          <size>
           <width>213</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>213</width>
           <height>23</height>
          </size>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>请输入密码</string>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::LogicalMoveStyle</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="password_3">
         <property name="text">
          <string> 再次输入密码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="confirm_password">
         <property name="minimumSize">
          <size>
           <width>213</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>213</width>
           <height>23</height>
          </size>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>请再次输入密码</string>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::LogicalMoveStyle</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="submit_register">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 233, 143);</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>300</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/back.jpg);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="user_login" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>351</width>
     <height>261</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>271</width>
      <height>119</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="user_form">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="id">
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="user_id">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>请输入用户名</string>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::LogicalMoveStyle</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="password">
         <property name="text">
          <string>密码  ：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="user_password">
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>请输入密码</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="submit_login">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 233, 143);</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
